plugin-server/src/utils/db/db.ts

```diff
@@ -1309,41 +1309,13 @@ export class DB {
         )
     }
 
-    // Event
+    // Event (NOTE: not a Django model, stored in ClickHouse table `events`)
 
     public async fetchEvents(): Promise<ClickHouseEvent[]> {
```

Comment:
This code should also be receiving tests as it's getting extensively rewritten. In this case the ClickHouseEvent types are subtly wrong (missing some columns) which would become apparent if this was tested.

Gonna skip this for now to avoid getting rabbit-holed but worth keeping in mind for the future.